x/protorev/keeper/keeper_test.go

```diff
@@ -846,16 +848,62 @@ func (suite *KeeperTestSuite) setUpPools() {
 			},
 			scalingFactors: []uint64{1, 1},
 		},
+		{ // Pool 46 - Used for epoch_hook UpdateHighestLiquidityPool testing
+			initialLiquidity: sdk.NewCoins(
+				sdk.NewCoin("epochOne", sdk.NewInt(1000)),
+				sdk.NewCoin("uosmo", sdk.NewInt(1000)),
+			),
+			poolParams: stableswap.PoolParams{
+				SwapFee: sdk.NewDecWithPrec(1, 4),
+				ExitFee: sdk.NewDecWithPrec(0, 2),
+			},
+			scalingFactors: []uint64{1, 1},
+		},
+		{ // Pool 47 - Used for epoch_hook UpdateHighestLiquidityPool testing
+			initialLiquidity: sdk.NewCoins(
+				sdk.NewCoin("epochOne", sdk.NewInt(1000)),
+				sdk.NewCoin("uosmo", sdk.NewInt(2000)),
+			),
+			poolParams: stableswap.PoolParams{
+				SwapFee: sdk.NewDecWithPrec(1, 4),
+				ExitFee: sdk.NewDecWithPrec(0, 2),
+			},
+			scalingFactors: []uint64{1, 1},
+		},
+		{ // Pool 48 - Used for epoch_hook UpdateHighestLiquidityPool testing
+			initialLiquidity: sdk.NewCoins(
+				sdk.NewCoin("epochTwo", sdk.NewInt(1000)),
+				sdk.NewCoin("uosmo", sdk.NewInt(1000)),
+			),
+			poolParams: stableswap.PoolParams{
+				SwapFee: sdk.NewDecWithPrec(1, 4),
+				ExitFee: sdk.NewDecWithPrec(0, 2),
+			},
+			scalingFactors: []uint64{1, 1},
+		},
 	}
 
 	for _, pool := range suite.stableSwapPools {
 		suite.createStableswapPool(pool.initialLiquidity, pool.poolParams, pool.scalingFactors)
 	}
 
+	// Create a concentrated liquidity pool for epoch_hook testing
+	suite.createConcentratedLiquiditiyPools()
+
 	// Set all of the pool info into the stores
 	suite.App.ProtoRevKeeper.UpdatePools(suite.Ctx)
 }
 
+func (suite *KeeperTestSuite) createConcentratedLiquiditiyPools() {
+	// Create a concentrated liquidity pool for epoch_hook testing
+	clPoolOne := suite.PrepareConcentratedPoolWithCoins("epochTwo", "uosmo")
+
+	// Provide liquidity to the pool
+	clPoolOneLiquidity := sdk.NewCoins(sdk.NewCoin("epochTwo", sdk.NewInt(1000)), sdk.NewCoin("uosmo", sdk.NewInt(2000)))
+	err := suite.App.BankKeeper.SendCoins(suite.Ctx, suite.TestAccs[0], clPoolOne.GetAddress(), clPoolOneLiquidity)
+	suite.Require().NoError(err)
+}
+
```

Comment:
nit: we probably don't need an entirely new function for this logic, and can just place this in setUpPools itself now.

Reply:
Sounds good, fixed: https://github.com/osmosis-labs/osmosis/pull/4946/commits/eb038cf9c94920f2b36ef2e81a5887a6fe046929